CIP-1853/README.md

```diff
@@ -53,8 +57,18 @@ Each stake pool is supposed to be managed separately so there is currently no in
 
 We chose hardened derivation at the usecase index as there is no incentive to mix the stake pool cold keys with other potential usecases and if there was such incentive, it would most likely be more appropriate to create a separate usecase/purpose for that.
 
+## Path to Active
+
+### Acceptance Criteria
+
+- [x] Standardisation of this derivation path among all wallets as of the Shelley ledger era.
```

Comment:
```suggestion
- [x] Standardisation of this derivation path among three wallets as of the Shelley ledger era.
```
I don't believe this is supported by many wallets/tools
I cant find any examples

Reply:
@Ryun1 my apologies; I must have misunderstood because I thought the Shelley stake pool key design was originally implemented like this (by my own assumption based on how `cardano-cli` was popularly used) & therefore "all wallets" would have supported it.

I'll leave it to you to submit any back-labelling to `Proposed` once confirmed... or will re-write it as such once you can confirm it.

Reply:
... OK, I was looking for my source material for marking this up as "universally implemented" and just found it (also applying to CIP-1855): https://github.com/cardano-foundation/CIPs/pull/729#issuecomment-1873538406

Reply:
OK, then that last paragraph of https://github.com/cardano-foundation/CIPs/pull/256#issue-1232625012 isn't strictly true.  I agree that 3 instances of support would therefore be a good measure of adoption.

Reply:
> OK, then that last paragraph of https://github.com/cardano-foundation/CIPs/pull/256#issue-1232625012 isn't strictly true.

IMO it is not true

Reply:
Thanks @Ryun1 for the explicit confirmation & I will also assume @Crypto2099 is aware of this by practical experience.